experimental/ExtendTeamsforM365/client/pages/categories.html

```diff
@@ -6,6 +6,15 @@
     <title>Northwind Product Categories</title>
     <link rel="stylesheet" href="/northwind.css" />
     <link rel="icon" href="data:;base64,="> <!-- Suppress favicon error -->
+    <style>               
+        #categoriesImgae{        
```

Comment:
We already have categoryImage class, let's use that instead,

Reply:
We changed the naming convention; it was used for ID purpose only.